opm/autodiff/BlackoilPropsAdFromDeck.hpp

```diff
@@ -59,6 +59,24 @@ namespace Opm
     {
         friend class BlackoilPropsDataHandle;
     public:
+        typedef typename SaturationPropsFromDeck::MaterialLawManager MaterialLawManager;
+
+        /// Constructor wrapping an opm-core black oil interface.
+        BlackoilPropsAdFromDeck(Opm::DeckConstPtr deck,
+                                Opm::EclipseStateConstPtr eclState,
+                                std::shared_ptr<MaterialLawManager> materialLawManager,
```

Comment:
yes. but that is the the domain of the time-stepper and thus same as in the current master. Even though I think that from a physical POV it is "more right" to update after each substep, the update after each report step arguably not wrong...